### DISCUSSION TITLE
@ghost should probibly not be recomended

### DISCUSSION THREAD

**Discussion by trickypr** [community/community#20362] [Other Feature Feedback, Questions, & Ideas] [answered]
I am getting recommendations to follow @ghost (the user that takes the place of deleted accounts). I personally think this is a special case that should not be allowed, given it is largely useless. 

![image](https://user-images.githubusercontent.com/23250792/178407589-93819399-e5e5-45e5-8c6a-2a18be89b98a.png)

**Comment by NellyisDevv** [comment]
I didn't even know that existed until today thats kind of funny, but maybe someone deleted their account so recently that they deleted their account before the recommended refreshed.

**Comment by siminapasat** [comment, marked as answer]
This shouldn't happen. Thanks for reporting it, we'll have a look!
